extensions/sql-bindings/src/contracts/azureFunctions/azureFunctionsContracts.ts

```diff
@@ -19,3 +19,27 @@ export namespace AddSqlBindingRequest {
 export namespace GetAzureFunctionsRequest {
 	export const type = new RequestType<GetAzureFunctionsParams, GetAzureFunctionsResult, void, void>('azureFunctions/getAzureFunctions');
 }
+
+// ------------------------------- < Execute String > ------------------------------------
+export interface SimpleExecuteParams {
+	queryString: string;
+	ownerUri: string;
+}
+
+/**
+ * Simple Query Execute Result will return rowCount, columnInfo, and rows from STS request
+ * rowCount is the number of rows returned with resultset
+ * columnInfo is the details about the columns that are povided as solutions
+ * rows is a 2D array of the cell values from the resultset
+ */
+export interface SimpleExecuteResult {
+	rowCount: number;
+	columnInfo: any;
+	rows: any;
+}
+
+export namespace SimpleExecuteRequest {
```

Comment:
Provide a link to https://github.com/microsoft/sqlops-dataprotocolclient/blob/main/src/protocol.ts#L437